src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] Please add categories reducer as well, currently categories reducer file is empty, it should also have an action and reducer, which should return `under construction`

- [ ] Categories component is missing, please add a category component, The component should contain:
- A button for checking the status

  -  this button would use a function checkCategoriesStatus (name can be different) made in the component (the name may be different)

   - checkCategoriesStatus  dispatches the underConstruction action creator (name of the function can be different)